src/Umbraco.Headless.Client.Net/Delivery/ContentDelivery.cs

```diff
@@ -124,9 +124,10 @@ public async Task<PagedContent> GetByType(string contentType, string culture = n
 
         public async Task<PagedContent<T>> GetByTypeAlias<T>(string contentTypeAlias, string culture = null, int page = 1, int pageSize = 10) where T : IContent
         {
-            var contentType = GetAliasFromClassName<T>();
+            if (string.IsNullOrWhiteSpace(contentTypeAlias))
+                throw new ArgumentException($"{nameof(contentTypeAlias)} cannot be null, empty or white space", contentTypeAlias);
```

Comment:
I think you meant pass `nameof(contentTypeAlias)` as the second parameter, since `contentTypeAlias` is always `null/empty` when we get here, and it expects the argument name as the parameter

```suggestion
                throw new ArgumentException("Value cannot be null or whitespace.", nameof(contentTypeAlias));
```